bundle/manifests/self-node-remediation.clusterserviceversion.yaml

```diff
@@ -380,6 +380,18 @@ spec:
                     memory: 110Mi
                 securityContext:
                   allowPrivilegeEscalation: false
+              - args:
+                - --secure-listen-address=0.0.0.0:8443
+                - --http2-disable
+                - --upstream=http://127.0.0.1:8080/
+                - --logtostderr=true
+                - --v=10
+                image: quay.io/brancz/kube-rbac-proxy:v0.15.0
+                name: kube-rbac-proxy
+                ports:
+                - containerPort: 8443
+                  name: https
+                resources: {}
```

Comment:
Can you please update the patch file for setting `resources` and `securityContext` like here? https://github.com/medik8s/node-healthcheck-operator/blob/main/config/default/manager_auth_proxy_patch.yaml#L23-L34

Reply:
Whoops

This is done, thank you!